src/main/web/templates/handlebars/list/partials/paginator.handlebars

```diff
@@ -10,9 +10,9 @@
             {{#each paginator.pages}}
             <li class="margin-top-sm--0 margin-top-md--0">
                 {{#if_eq this paginator.currentPage}}
-                <span class="page-link btn btn--plain btn--plain-active btn--block">{{this}}</span>
+                <span class="page-link btn btn--plain btn--plain-active btn--block" aria-current="true" aria-label="Page {{this}} of {{paginator.numberOfPages}}">{{this}}</span>
```

Comment:
Can this be made a `<a>` please as it is not keyboard navigable and can be missed?